test/fw/bin/pbs_benchpress

```diff
@@ -340,6 +348,42 @@ if __name__ == '__main__':
             'Testsuites names require (see -t) along with -i option!')
         sys.exit(1)
 
+    try:
+        tc_failure_threshold = int(tc_failure_threshold)
+        if tc_failure_threshold < 0:
+            raise ValueError
+    except:
```

Comment:
@hirenvadalia Thanks for the explanation, I totally overlooked the typecast that you are doing there.